.github/workflows/on_pull_request.yaml

```diff
@@ -0,0 +1,30 @@
+name: ⬣ Pull Request
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    name: ⬣ Linters
+    runs-on: ubuntu-24.04
```

Comment:
I'm not using `-latest` because I've already encountered issues when the latest version changed the target Ubuntu release, and some dependencies in the cache stopped working across different Ubuntu versions.